esmtools/stats.py

```diff
@@ -301,3 +301,12 @@ def ACF(ds, dim='time', nlags=None):
         acf.append(res)
     acf = xr.concat(acf, dim=dim)
     return acf
+
+
+def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2):
+    """Parallelize scipy.stats.ttest_ind_from_stats."""
+    return xr.apply_ufunc(tti_from_stats, mean1, std1, nobs1, mean2, std2,
+                          nobs2,
+                          input_core_dims=[[], [], [], [], [], []],
+                          output_core_dims=[[], []],
+                          vectorize=True, dask='parallelized')
```

Comment:
general comment: I am unsure whether parallelized is really a good idea.
Tip on http://xarray.pydata.org/en/stable/dask.html suggests:
> For the majority of NumPy functions that are already wrapped by Dask, it’s usually a better idea to use the pre-existing dask.array function, by using either a pre-existing xarray methods or apply_ufunc() with dask='allowed'. Dask can often have a more efficient implementation that makes use of the specialized structure of a problem, unlike the generic speedups offered by dask='parallelized'.

Reply:
I will have to investigate this for xskillscore in detail. I think it doesnt really change much, but I cannot prove it yet

Reply:
I can try running this on CESM data with `parallelized` and `allowed` and see timing differences. The issue is a dask array can't be put through the function because earlier lines of code (before the t-test) require it to be loaded into memory.